frontend/public/components/utils/status-icon.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { Icon } from 'patternfly-react';
```

Comment:
We should check that this doesn't significantly increase the size of our main bundle.